src/invidious/views/error.ecr

```diff
@@ -1,8 +1,23 @@
+<%
+  dark_mode = env.get("preferences").as(Preferences).dark_mode
+%>
+
 <% content_for "header" do %>
 <title><%= "Error" %> - Invidious</title>
 <% end %>
 
 <div class="h-box">
-    <%= error_message %>
+    <div style="display: flex; flex-direction: column; align-items: center;">
+        <%= error_message %>
+        <% if dark_mode == "dark" %>
+            <div style="width: 100px; height: auto;">
+                <img src="/404_tv_box_dark_theme.svg" alt="SVG Image" style="width: 100px; height: auto;">
```

Comment:
An empty alt should be used instead unless there's information in the image that should be conveyed to users that use a screenreader.
```suggestion
                <img src="/404_tv_box_dark_theme.svg" alt="" style="width: 100px; height: auto;">
```